### DISCUSSION TITLE
I sincerely want to know how to install it on linux system thanks!

### DISCUSSION THREAD

**Discussion by fanke12345** [libvips/libvips#3666] [Q&A]
import _libvips
ModuleNotFoundError: No module named '_libvips'

**Comment by jcupitt** [comment]
You'd need to say what linux system you are installing on, but on Ubuntu it would be:

```
sudo apt install libvips
```

To install the library.

If you want to use libvips from python, you need:

```
sudo apt install libvips-dev
pip3 install pyvips
```

Then:

```python
import pyips
```
